### PR TITLE
feat(styling) : [0001] css styling modification

### PR DESCRIPTION
**Pull request by sandhrarahardjo**
**What problem does this PR solve?**
The lack of styling on the page

**How does this PR solve it?**
Adding columns and background colours 

**Screenshots :**
Screenshot (109)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-
+import './App.css';
 const root = ReactDOM.createRoot(document.getElementById('root'));
```
Comment (andreantaufik) [inline src/index.js:6]: add new line after import.

**Review by andreantaufik — CHANGES_REQUESTED** [review]
minor review

**Comment by andreantaufik** [comment]
// import from external source
import React from 'react';
import ReactDOM from 'react-dom/client';

// local source
import App from './App';
import reportWebVitals from './reportWebVitals';
import './App.css';

// codee

**Comment by sandhrarahardjo** [comment]
fix(minor changes) : [0001] minor import order fixed

**What problem does this PR solve?**
Import order of the code

**How does this PR solve it?**
Fixed the import order of the code

**Screenshot : **
Screenshot (111)

**Review by andreantaufik — APPROVED** [review]

**Pull request merged**